src/main/java/com/magnaboy/Util.java

```diff
@@ -71,4 +72,27 @@ public static String intArrayToString(int[] array) {
 		}
 		return sb.toString().replaceAll("\\s+", "");
 	}
+
+	public static boolean samePosition(WorldPoint p1, WorldPoint p2)
+	{
+		return p1.getX() == p2.getX() &&
+			p1.getY() == p2.getY() &&
+			p1.getPlane() == p2.getPlane();
+	}
```

Comment:
Do we need this? instead of `WorldPoint.equals`?

Reply:
I was unaware of that method. We do not need this method then. I will scrap it.

Reply:
Fixed